packages/manager/modules/bm-server-components/src/network-interfaces/rename/interfaces-rename.module.js

```diff
@@ -1,11 +1,9 @@
 import component from './interfaces-rename.component';
-import routing from './interfaces-rename.routing';
 
 const moduleName = 'ovhManagerDedicatedServerInterfacesRename';
 
 angular
-  .module(moduleName, ['ui.router'])
-  .config(routing)
+  .module(moduleName, ['ovh-api-services'])
```

Comment:
same https://github.com/ovh/manager/pull/10018#discussion_r1334512760

Reply:
Yes the controller uses `OvhApiDedicatedServerVirtualInterface`